src/main.js

```diff
@@ -29,6 +29,8 @@ import { sync } from 'vuex-router-sync'
 /** Global directives */
 import VTooltip from '@nextcloud/vue/dist/Directives/Tooltip.js'
 
+__webpack_public_path__ = OC.linkTo('collectives', 'js/') // eslint-disable-line
```

Comment:
Sorry I copied the outdated one, we should rather use the npm package API:

```js
import { generateFilePath } from '@nextcloud/router'

__webpack_public_path__ = generateFilePath('collectives', '', 'js/')
```

Reply:
Oh, thanks for the pointer. Fixed now!